koans/TrueAndFalseKoans.php

```diff
@@ -7,5 +7,105 @@
 
 class TrueAndFalseKoans extends TestCase
 {
-    // TODO: Add tests for == and ===, != and !== (FALSE, null, 0, "")
+    // Rule: You can only use true or false for the solution
+
+    /**
+     * @testdox 3 and '3' are the same thing, but, the type? (Equal operator)
```

Comment:
Like the other PR, change the tests names and delete the comment.

Reply:
And wouldn't it be interesting to leave the comments due to the fact that if they are removed when executing the tests from the terminal, it would not give us any information about that test? I leave attached a photo in which you can see an example in which the first two tests have no comments and the following do.

![imagen](https://github.com/540/php_koans/assets/113472661/ce92e1c5-6753-43a3-8ab7-234a0c5c1040)